docs/source/services/timelock_service/installation.rst

```diff
@@ -18,10 +28,20 @@ Timelock Server Installation
   This will place a tar file in the ``build/distributions`` directory of the ``timelock-server`` project. The
   tar file follows semantic versioning.
 
-3. Copy this tar file to the server on which you want to deploy Timelock Server.
-   (If you wish to run Timelock Server locally, this step may be omitted.)
+Install Timelock (without clients)
+----------------------------------
+
+We recommend deploying either a 3-node or a 5-node Timelock cluster.
```

Comment:
Might be obvious, but I think it could be useful to give a bit of guidance on how to choose between these configurations (we can link to http://palantir.github.io/atlasdb/html/services/lock_service/installation.html, which is for distributed lock-service but also makes sense for timelock). Not a blocker - if it's too much work don't bother